Sources/Meta/Extensions/String+NonOptional.swift

```diff
@@ -0,0 +1,14 @@
+extension String {
+    var nonOptional: String {
+        substring(pattern: "(?<=Optional<).*(?=>)")
+            .substring(pattern: "(?<=Option<).*(?=>)")
+            .clean("?")
+            .trimmingCharacters
+    }
+    
+    var isOptionalType: Bool {
+        contains("?") ||
+        contains("Optional<") ||
+        contains("Option<")
+    }
```

Comment:
I think we will need to revisit this logic, as something of type `Foo<Int?>` will have `isOptionalType == true`, but that is not correct.